code/modules/clothing/spacesuits/rig/rig_attackby.dm

```diff
@@ -1,24 +1,25 @@
 /obj/item/rig/attackby(obj/item/attacking_item, mob/user)
 
-	if(!istype(user,/mob/living)) return 0
+	if(!isliving(user))
+		return FALSE
 
-	if(electrified != 0)
+	if(electrified > 0)
```

Comment:
Have you checked if -1 is used as "permanently electrified"?

Reply:
Hm you might be right, though there's no documentation about it and we disallow magic numbers, either way i'll restore it to how it was